active/tab/tabcontrol.md

```diff
@@ -0,0 +1,245 @@
+
+    // Set the Window's content to the new page
+    ElementCompositionPreview.SetAppWindowContent(newWindow, newPage);
+
+    // Show the window
+    await newWindow.TryShowAsync();
+}
+```
+
+## Databind to a set of tabs 
+
+``` xml
+<TabControl ItemsSource="{x:Bind TabItemCollection}" />
+```
+
+# Remarks
+<!-- Explanation and guidance that doesn't fit into the Examples
+section.  For example, see the Remarks for the MediaPlayerElement 
+(https://docs.microsoft.com/uwp/api/Windows.UI.Xaml.Controls.MediaPlayerElement#remarks). -->
+
+## Keyboarding behavior
+### TAB and Arrow key behavior
+The TabControl has areas containing left content, right content, and the Tab content (containing 0 or more Tabs). When the user hits the TAB key, focus moves between the content areas. When focus moves into the Tab content area, the selected Tab gains focus. Once the user is in the Tab content area, pressing the arrow key will move focus between the tabs. Arrow focus is trapped inside the Tab content area.
+
+### Selecting a Tab
+When a Tab has focus, pressing SPACE or ENTER will select that Tab.
+
+### Shortcuts for moving between tabs
+Ctrl+TAB will select the next Tab.
+Ctrl+Shift+TAB will select the previous Tab.
+Tab selection cycles (meaning if the user has selected the last tab and presses Ctrl+TAB, the first tab becomes selected).
+
+Ctrl+1 through 8 will automatically select the corresponding Tab.
+Ctrl+9 will automatically select the last Tab.
+
+### Closing a Tab 
+Hitting CTRL+W or CTRL+F4 will automatically close the selected Tab (assuming it is closable). 
+
+### Keyboard guidance for App Developers
+The above sections outline built-in keyboarding behavior provided by the TabControl. However, there are certain expected keyboard shortcuts that you will be responsible for implementing. 
+* If your Tab control supports adding a new Tab, users expect CTRL+T to open a new tab
+* Consider maintaining a list of recently closed Tabs. If the user presses CTRL+SHIFT+T, they expect recently closed tabs to be reopened.
+* If users can perform more commands on a Tab than just closing the Tab (for example, pinning a Tab or duplicating a Tab), consider adding a context menu to the Tab. 
+
+# API Notes
+<!-- Give a one or two line description of each API (type
+and member), or at least the ones that aren't obvious
+from their name.  These descriptions are what show up
+in IntelliSense. -->
+
+### TabControl properties and events
+
+| Property | Type | Description |
+|:-------- |:---- |:----------- |
+| CanCloseTabs | bool | Default value for the item if it doesn't specify a IsClosable value. |
+| CloseButtonOverlay | enum | Describes the behavior of the close button. Values are {Auto, OnHover, Persistent} |
+| ItemHeaderTemplate | DataTemplate | Default template for the item if no template specified. |
+| SelectedTabWidth | double | The size of the selected tab header. |
+| TabHeader | object | Content to the left of the tab strip. |
```

Comment:
Maybe it's just me, but the TabHeader and ItemHeader(Template) concepts sound very similar and potentially confusing. I know you're proposing names that are independent of reading order. But as I've learned from NavigationView, header gets overloaded and overused very quickly.

Reply:
There are a lot of "Headers", I agree. This largely comes from WPF, which had TabControl and TabItem and their corresponding Header properties as well. 

I'm open to suggestions here. One consideration is the discussion about "TabStripPlacement" which could support Tabs on the left. We're not supporting placement for the time being, but I do want to avoid calling the content "LeftTabContent/RightTabContent" because that won't make sense if we do support TabStripPlacement.

Reply:
How about using prefix/suffix terminology? Something like calling the TabHeader property TabsPrefix or TabStripPrefix, and similar with Suffix instead of Footer.